packages/plugin-analytics/src/utils/PlausibleAnalytics.ts

```diff
@@ -0,0 +1,100 @@
+import { logger } from 'robo.js'
+import { BaseEngine, EventOptions, ViewOptions } from './analytics'
```

Comment:
Don't forget to include the `.js` suffix in imports. TypeScript still has issues with this.

Reply:
Oh, and `import type` whenever you can please.